pygmt/src/solar.py

```diff
@@ -0,0 +1,98 @@
+"""
+solar - Plot day-night terminators.
+"""
+
+import datetime
+
+import pandas as pd
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+
+
+@fmt_docstring
+@use_alias(
+    R="region",
+    J="projection",
+    G="fill",
+    B="frame",
+    W="pen",
+    U="timestamp",
+    V="verbose",
+    X="xshift",
+    Y="yshift",
+    c="panel",
+    p="perspective",
+    t="transparency",
+)
+@kwargs_to_strings(R="sequence", p="sequence")
+def solar(self, terminator="d", terminator_datetime=None, **kwargs):
+    r"""
+    Plot day-light terminators.
+
+    This function plots the day-night terminator. Additionally, it can
+    plot the terminators for civil twilight, nautical twilight, and
+    astronomical twilight.
+
+    {aliases}
+
+    Parameters
+    ----------
+    terminator : str
+        Set the type of terminator displayed. Valid arguments are
+        **day_night**, **civil**, **nautical**, and **astronomical**, which
+        can be set with either the full name or the first letter of the name.
+        [Default is ``day_night``]
+    terminator_datetime : str or datetime object
+        Set the UTC date and time of the displayed terminator. It can be
+        passed as a string, using the format *YYYY-MM-DD HH:MM:SS*
+        (e.g. 0600 on January 1, 2000 would be ``2000-01-01 06:00:00``). A
+        datetime object can be passed as well.
+        [Default is the current date and time]
```

Comment:
Two comments here:

- Default date and time is in UTC time
- `pd.to_datetime` support many different datetime formats.

Reply:
I'm not sure what you're asking me to change here

Reply:
The description of the `terminator_datetime` is a little misleading. `pd.to_datetime` can convert many datetime formats (e.g., `2001-01-01 00:00:00`, `2001/1/1`) to a datetime, but the description reads like only *YYYY-MM-DD HH:MM:SS* is allowed.

> [Default is the current date and time]

The default date and time is in UTC time, but the description sounds like "in local time".

Reply:
Updated in 2e5e151 and 631b24d